go/vm/geth/geth.go

```diff
@@ -126,6 +137,11 @@ func createGethInterpreterContext(parameters vm.Parameters) (*geth.EVM, *geth.Co
 		Transfer:    transferFunc,
 		CanTransfer: canTransferFunc,
 	}
+
+	if parameters.Revision >= vm.R11_Paris {
+		blockCtx.Random = &common.Hash{}
```

Comment:
This field may be the input for the `PREVRANDO` operation in geth. If so, `PREVRANDO` should fail in Paris.

To fix, set the value to the same value as the `Difficulty` in line 158.

Reply:
I checked the vm, there is a new function for PREVRANDAO which fetches the value from this new context field. 
I update them to use the same value.